README.md

```diff
@@ -1159,9 +1159,9 @@ client.mutate(transaction)
 
 An important note on this approach is that you cannot call `commit()` on transactions or patches instantiated this way, instead you have to pass them to `client.mutate()`
 
-### Server Side Actions
+### Actions
 
-The Server Side Actions API provides a new interface for creating, updating and publishing documents. It is a wrapper around the [Server Side Actions API](https://www.sanity.io/docs/http-actions).
+The Actions API provides a new interface for creating, updating and publishing documents. It is a wrapper around the [Server Side Actions API](https://www.sanity.io/docs/http-actions).
```

Comment:
```suggestion
The Actions API provides a new interface for creating, updating and publishing documents. It is a wrapper around the [Actions API](https://www.sanity.io/docs/http-actions).
```

Reply:
I was wondering about that, I'm not sure it should be changed, as that's actually the server side actions api?

Reply:
That's what we renamed a few weeks ago. If you click through you'll find that it is just called "Actions API".